html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1> Coding Form!</h1>
+    <form>
+      <label for="email">
+        Email
+        <input type="text" name="email"/><br>
+      </label>
+      <label for="dropdown">
+        Why is Coding cool?<br>
+        <select name="dropdown">
+          <option value="select an option">Select an option</option>
+          <option value="because it's interesting">Because it's interesting</option>
+          <option value="because it's fun">Because it's fun</option>
+          <option value="because it's powerful">Because it's powerful</option>
+        </select><br>
+      </label>
+      <label for="favorite-language">
+        Favorite Language (choose one)<br>
+        <input type="radio" name="language" value="HTML"/> HTML
+        <input type="radio" name="language" value="CSS"/> CSS
+        <input type="radio" name="language" value="Javascript"/> JavaScript <br>
+      </label>
```

Comment:
Each of these `input` elements should have it's own `label`, rather than wrapping the whole thing in 1 `label`.